### ISSUE TITLE
Source URI changed?

### ISSUE THREAD

**Issue by sikha-root** [hrehfeld/archlinux-whisper.cpp-model#3]
Attempting to install any of the models leads to the following error:

```
==> Making package: whisper.cpp-model-large 1-1 (Sun 09 Jul 2023 10:53:54 PM JST)
==> Retrieving sources...
  -> Downloading large.bin...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0    29    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (22) The requested URL returned error: 401
==> ERROR: Failure while downloading https://huggingface.co/datasets/ggerganov/whisper.cpp/resolve/main/ggml-large.bin
    Aborting...
error: failed to download sources for 'whisper.cpp-model-large-1-1':
error: packages failed to build: whisper.cpp-model-large-1-1
```

At first I thought huggingface was blocking my cloud IP but it looks like there is HTTP basic auth on these endpoints. Removing `datasets` from the URI, however, appears to lead to the model files and I was able just update the URI in the PKGBUILD to get it to install, so maybe it just changed?

**Comment by mariuszste** [comment]
yep, here is the relevant commit on the upstream downloader: https://github.com/ggerganov/whisper.cpp/commit/1beff6f66d2a9e293c08b736e5f14a42ff3922d1#diff-d397dc66c645650b92fbd72dce0ccd798a569bdd071cbd332079d70e6312d12a

**Comment by hrehfeld** [comment]
fixed by this awesome PR #4
